test/powershell/Modules/Microsoft.PowerShell.Utility/Pester.Commands.Cmdlets.Json.Tests.ps1

```diff
@@ -1472,7 +1489,7 @@ Describe "Json Bug fixes"  -Tags "Feature" {
 
     It "ConvertFrom-Json deserializes an array of strings (in multiple lines) as a single string." {
 
-        $result = "[1,","2,","3]" | ConvertFrom-Json
+        $result = "[1,","2,","3]" | ConvertFrom-Json
```

Comment:
Same reply as above. I did not make these changes, and believe them to be due to inconsistent newlines. My recommendation is to accept the change to make newlines consistent so that future changes in this file don't run into the same issue.